roles/repositories/README.md

```diff
@@ -0,0 +1,108 @@
+theforeman.foreman.repositories
+===============================
+
+This role defines Products and enabled Red Hat Repositories and Custom Repositories.
```

Comment:
Nit: `defines Products, enabled Red Hat Repositories and Custom Repositories.`

Reply:
Should this rather be "This role defines Products and Custom Repositories and enables Red Hat Repositories."
As it does not *enable* custom repos?

Reply:
Yes, I think so. I'm also leaning to renaming the role `Products` with the shape it is taking